server/Cargo.toml

```diff
@@ -47,6 +47,7 @@ uuid = "1.0.0"
 base64 = "0.13"
 
 [dev-dependencies]
+crucible = { git = "https://github.com/oxidecomputer/crucible", rev = "8314eeddd228ec0d76cefa40c4a41d3e2611ac18" }
```

Comment:
Why the specific version dep?

Reply:
I chose this since it's the same version propolis and propolis-client are using, but I ended up resolving this by re-exporting the `VolumeConstructionRequest` type (the reason server needs this) from the instance spec library itself and not adding this dependency at all. (The semantics I really want are "the same VolumeConstructionRequest definition that the spec library used," and from what I could gather the `pub use` re-export was the most idiomatic way to achieve that.)